example/platform.html

```diff
@@ -97,8 +97,9 @@
 <script>
   function initViewer() {
     var viewerHost = document.getElementById('viewerHost');
-    var ampUrl = 'https://www.ampproject.org/';
-    var viewer = new AmpViewer(viewerHost, ampUrl);
+    var ampDocUrl = 'www.ampproject.org';
```

Comment:
I would suggest just passing in the ampDocUrl and then computing the ampDocCachedUrl from it. The algorithm to compute the subdomain is non-trivial so it seems reasonable to leave this as a TODO, if you want.